src/together/cli/api/finetune.py

```diff
@@ -200,6 +200,19 @@ def fine_tuning(ctx: click.Context) -> None:
     "The format: {$JOB_ID/$OUTPUT_MODEL_NAME}:{$STEP}. "
     "The step value is optional, without it the final checkpoint will be used.",
 )
+@click.option(
+    "--from-hf-model",
+    type=str,
+    default=None,
+    help="The Hugging Face repo to start training from. "
+    "Should be paired with the base model, specified in `model` argument.",
```

Comment:
By the way, "paired with" is not 100% clear. Does it mean that the model from the Hub should use the same architecture?